### PR TITLE
fix: UI fixes and improvements

### PR DESCRIPTION
**Pull request by Udit-takkar**
- Added cursor pointer

https://github.com/documenso/documenso/assets/53316345/ad6cc842-bf3e-4b82-a873-6afee34cdeab

- Founder section in dark mode 

BEFORE:-

image

AFTER:-

Screenshot 2023-10-14 at 7 18 43 PM

- Tooltip label color not visible in dark mode 

Before:-

Screenshot 2023-10-14 at 7 28 25 PM

After:-
Screenshot 2023-10-14 at 7 30 20 PM

- Admin Data Table Owner name not readable

Before

image

After:-  ( Added tooltip and shortened the name)

Screenshot 2023-10-14 at 7 42 16 PM

- Mobile UI issues in admin stats page 

BEFORE:-

Screenshot 2023-10-14 at 8 36 22 PM

svg Icon size  scaled down in mobile

Screenshot 2023-10-14 at 10 42 10 PM

Screenshot 2023-10-14 at 10 43 44 PM

AFTER:-

https://github.com/documenso/documenso/assets/53316345/244e4c29-eafc-40ca-a4cb-a1362dba26f8

Screenshot 2023-10-14 at 10 43 08 PM

Screenshot 2023-10-14 at 10 43 26 PM

### CODE REVIEW

**Comment by Mythie** [comment]
Hey there, it looks like you haven't accepted our contributor license agreement yet. In order for us to accept your pull request we ask that you please fill out the CLA:

https://documen.so/cla

**Comment by dguyen** [comment]
Scroll bar change appears to be already being worked:

https://github.com/documenso/documenso/issues/441

**Comment by Udit-takkar** [comment]
@dguyen ok thanks removed it from my PR

File: apps/web/src/app/(dashboard)/admin/documents/data-table.tsx [hunk 1]
```diff
@@ -58,14 +60,24 @@ export const DocumentsDataTable = ({ results }: DocumentsDataTableProps) => {
...
+                        <AvatarFallback className="text-gray-400">
+                          <span className="text-xs">{avatarFallback}</span>
+                        </AvatarFallback>
+                      </Avatar>
+                    </Link>
+                  </TooltipTrigger>
+                  <TooltipContent className="max-w-xs">
+                    {row.original.User.name ?? row.original.User.email}
+                  </TooltipContent>
+                </Tooltip>
```
Comment (dguyen) [inline apps/web/src/app/(dashboard)/admin/documents/data-table.tsx:80]: Can you make it look something like this?

image

Hover on the avatars on the `/documents` to see a similar design

**Comment by catalinpit** [comment]
Hey @Udit-takkar, great work! Just a heads up that I opened a quick PR (https://github.com/documenso/documenso/pull/570) to fix the user avatar issue. It's a small fix and you can continue with the work on the tooltip.

**Comment by Udit-takkar** [comment]
@dguyen Done. Addressed all the comments

**Comment by Udit-takkar** [comment]
@dguyen Done. Addressed all the comments

**Comment by dguyen** [comment]
The mobile screen for admin panel doesn't seem to work well with larger mobile devices like this:

image

**Comment by Udit-takkar** [comment]
@dguyen fixed for large mobile devices

https://github.com/documenso/documenso/assets/53316345/52dbe1d1-3952-42d9-828c-66724211eab2

**Comment by catalinpit** [comment]
> @dguyen fixed for large mobile devices
> 
>  Screen.Recording.2023-10-17.at.1.01.08.PM.mov

Just throwing out ideas for discussions, but is the horizontal scroll preferred for the nav buttons? Should we have them in a column if the screen size is too small or something similar?

Screenshot 2023-10-17 at 11 41 23

**Comment by Udit-takkar** [comment]
I think Horizontal scroll is fine. It's used in other web applications also

Screenshot 2023-10-18 at 3 18 58 AM

**Comment by adithyaakrishna** [comment]
Just adding my 2 cents, we also use Horizontal Scrolling for tabs at Red Hat. It kinda makes the overall feel on mobile devices a bit more polished and improves UX is what we feel

**Comment by catalinpit** [comment]
> I think Horizontal scroll is fine. It's used in other web applications also
> 
> Screenshot 2023-10-18 at 3 18 58 AM

Oky doky. Then leave it like that. 👍😃

**Comment by Udit-takkar** [comment]
https://github.com/documenso/documenso/assets/53316345/e0466f49-061c-4ecc-9a4b-531c4e231550

@catalinpit okay added a no-scrollbar utility

**Comment by dguyen** [comment]
Sorry but we're going to need a horizontal scrollbar for this

On my phone I can't see any indication there's another horizontal element.

image

**Comment by Udit-takkar** [comment]
@dguyen removed it like before 👍

**Review by dguyen — APPROVED** [review]

**Comment by dguyen** [comment]
Thanks for those changes

/tip 40

**Pull request merged**